src/java.base/share/classes/java/util/Locale.java

```diff
@@ -258,6 +259,74 @@
  * locales. For example, {@code Locale.US} is the {@code Locale} object
  * for the United States.
  *
+ * <h3><a id="default_locale">Default Locale</a></h3>
+ *
+ * <p>The default Locale is provided mainly for the locale-sensitive methods if no
+ * {@code Locale} is explicitly specified as an argument, such as
+ * {@link DateFormat#getInstance()}. The default Locale is determined at startup
+ * of the Java runtime and established in the following three phases:
+ * <ol>
+ * <li>The locale-related system properties listed below are established from the
+ * host environment. Some system properties (except for {@code user.language}) may
```

Comment:
IIUC, should it be _all_ system properties, not _some_ if we are defining the single exception?

Reply:
The host-dependent part of the code always generates at least the language, thus it was excluded.